eng/pipelines/templates/scripts/create-tags-and-git-release.yml

```diff
@@ -0,0 +1,11 @@
+steps:
+- task: PowerShell@2
+  displayName: 'Verify Package Tags and Create Git Releases'
+  inputs:
+    targetType: filePath
+    filePath: scripts/powershell/create-tags-and-git-release.ps1
```

Comment:
We should consider moving this template as well as the script under eng/common directory, especially if we think we can potentially eliminate the need to pull any yml templates from the azure-sdk-tools repo. I'm not sure how feasible of a goal that is yet but something we should try and do if we can.

Reply:
This can be moved to `eng/common` as it is pretty generic. But `create-tags-and-git-release.ps1` which it calls is not language specific.

Reply:
I'm not sure I understand why you think the template can be moved by the ps1 script shouldn't. They are both support and are shared by all the languages, as in they are common. Sure the ps1 script might have some special logic per language but that OK. Eventually we should see if we can make them a little more neutral but while it needs to parse each of the package types it will likely need to have some specialized logic per language (assuming we cannot eliminate that need to parse the packages).

Reply:
Sure I can move it into `eng/common`. My thinking was that some part of the code is not needed for the other repos. But I'll go ahead and move it.